app/views/unattended/provisioning_templates/user_data/preseed_autoinstall_cloud_init.erb

```diff
@@ -67,9 +69,10 @@ autoinstall:
     allow-pw: true
     install-server: true
   updates: security
-<%= indent(2) { @host.diskLayout } -%>
+<%= indent(2) { @host.diskLayout } %>
 <%= indent(2) { snippet_if_exists(template_name + " custom root") } -%>
   late-commands:
+<%= indent(2) { snippet 'preseed_autoinstall_clevis_tang_wrapper' if host_param('disk_enc_tang_servers') and os_major >= 22 and os_minor >= 3 } %>
```

Comment:
This is ERB, so you use `&&`. That, and prevent empty lines
```suggestion
<%= indent(2) { snippet 'preseed_autoinstall_clevis_tang_wrapper' if host_param('disk_enc_tang_servers') && os_major >= 22 && os_minor >= 3 } -%>
```

Reply:
Regarding empty lines, please have a look at the discussion above* and tell me explicitly which way you guys prefer. As said, avoiding empty lines in ERB requires empty lines in the corresponding snippets.

* https://github.com/theforeman/foreman/pull/9887/files/0c41804d3774276216a151fd3f06f384fc7908d0#diff-02b11c45a8fe92a6ae919a699d0198e01b0f9fca7b3449725e6406ebae836fa3